src/lib/integration_test_lib/constants.ml

```diff
@@ -0,0 +1,13 @@
+let aws_region = "us-west-2"
+
```

Comment:
this file should be in /integration_test_cloud_engine because it has to do with the infra engine, and not the DSL and core logic of lucy.

Reply:
also these are more like defaults rather than constants no?  maybe rename the file infra_defaults.ml